tests/lib/tools/tests.info

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -eu
+
+show_help() {
+    echo "usage: tests.info <CMD>"
+    echo
+    echo "Supported commands:"
+    echo "    is-snapd-pkg-repo: indicates when the snap packages is from the repository"
```

Comment:
should this be called `is-snapd-deb-from-repo` since it only works for debs? or is there a missing TODO to make this work on other systems as well?

Reply:
well, if at some point the re-execution is implemented for rpms, then we could make other systems download the rpm as we do with the deb.

Reply:
I think the name is fine and should not include the word "deb" -- we will soon enable this for RPMs in tests. My only personal preference would be to call it "is-snapd-from-archive" as it reads better than abbreviated repository IMO. No need to change this though :)